<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLVolumeInfoWidget</class>
 <widget class="qMRMLWidget" name="qMRMLVolumeInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>421</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Volume Information</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="ImageDimensionsLabel">
     <property name="text">
      <string>Image Dimensions:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="ctkCoordinatesWidget" name="ImageDimensionsWidget">
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="decimalsOption">
      <set>ctkDoubleSpinBox::FixedDecimals</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="ImageSpacingLabel">
     <property name="text">
      <string>Image Spacing:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="qMRMLCoordinatesWidget" name="ImageSpacingWidget">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="decimalsOption">
      <set>ctkDoubleSpinBox::DecimalsByKey|ctkDoubleSpinBox::DecimalsByShortcuts|ctkDoubleSpinBox::DecimalsByValue</set>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
     <property name="coordinates" stdset="0">
      <string>0,0,0</string>
     </property>
     <property name="quantity">
      <string notr="true">length</string>
     </property>
     <property name="unitAwareProperties">
      <set>qMRMLCoordinatesWidget::Precision|qMRMLCoordinatesWidget::Prefix|qMRMLCoordinatesWidget::Scaling|qMRMLCoordinatesWidget::Suffix</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="ImageOriginLabel">
     <property name="text">
      <string>Image Origin:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="qMRMLCoordinatesWidget" name="ImageOriginWidget">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="decimalsOption">
      <set>ctkDoubleSpinBox::DecimalsByKey|ctkDoubleSpinBox::DecimalsByShortcuts|ctkDoubleSpinBox::DecimalsByValue</set>
     </property>
     <property name="minimum">
      <double>-1000000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
     <property name="quantity">
      <string notr="true">length</string>
     </property>
     <property name="unitAwareProperties">
      <set>qMRMLCoordinatesWidget::Precision|qMRMLCoordinatesWidget::Prefix|qMRMLCoordinatesWidget::Scaling|qMRMLCoordinatesWidget::Suffix</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="IJKToRASDirectionLabel">
     <property name="text">
      <string>IJK to RAS Direction Matrix:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ctkMatrixWidget" name="IJKToRASDirectionMatrixWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <property name="rowCount">
      <number>3</number>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="decimalsOption">
      <set>ctkDoubleSpinBox::DecimalsByKey|ctkDoubleSpinBox::DecimalsByShortcuts|ctkDoubleSpinBox::DecimalsByValue</set>
     </property>
     <property name="singleStep">
      <double>0.000100000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="CenterVolumePushButton">
     <property name="toolTip">
      <string>Set a parent transform to the volume that center it on the origin. Harden the transform to permanently change the volume position.</string>
     </property>
     <property name="text">
      <string>Center Volume</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="ScanOrderLabel">
     <property name="text">
      <string>Scan Order:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="ScanOrderValueLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ScanOrderComboBox"/>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="NumberOfScalarsLabel">
     <property name="text">
      <string>Number of Scalars:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="NumberOfScalarsValueLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="NumberOfScalarsSpinBox">
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="ScalarTypeLabel">
     <property name="text">
      <string>Scalar Type:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="ScalarTypeValueLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ScalarTypeComboBox"/>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="ScalarRangeLabel">
     <property name="text">
      <string>Scalar Range:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="ScalarRangeValueLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="VolumeTypeLabel">
     <property name="text">
      <string>Volume type:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="VolumeTagLabel">
     <property name="text">
      <string>VolumeTag</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="FileNameLabel">
     <property name="text">
      <string>File Name:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLineEdit" name="FileNameLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="WindowLevelPresetsLabel">
     <property name="text">
      <string>Window/Level Presets:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QListWidget" name="WindowLevelPresetsListWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLCoordinatesWidget</class>
   <extends>ctkCoordinatesWidget</extends>
   <header>qMRMLCoordinatesWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCoordinatesWidget</class>
   <extends>QWidget</extends>
   <header>ctkCoordinatesWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkMatrixWidget</class>
   <extends>QWidget</extends>
   <header>ctkMatrixWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qMRMLVolumeInfoWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ImageOriginWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qMRMLVolumeInfoWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ImageSpacingWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
